pkg/template/validation_test.go

```diff
@@ -9,11 +9,11 @@ import (
 	"github.com/Masterminds/semver/v3"
 )
 
-var releaseVersionsAWS1150 = Versions{
-	Calico:                       "3.10.4",
-	CRITools:                     "1.16.1",
+var releaseVersionsKVM1220 = Versions{
```

Comment:
This is kind of ugly. We need a realistic release to test validation. Let me know if you have ideas for improving it.

Reply:
Can we keep the variable name as e.g. `releaseVersion` and add a comment which explains that the versions are equivalent to some release.

Reply:
Good call. This is done now.